### ISSUE TITLE
Why sum(bytesWritten) => 0 (0 Bytes) always 0 ?

### ISSUE THREAD

**Issue by gilv** [LucaCanali/sparkMeasure#8]
I used Spark to read local Parquet file and store it under different name.
I used file:// as i accessed file locally.

sum(bytesWritten) => 0 (0 Bytes) was reported 0. Why is this?

Thanks

**Comment by LucaCanali** [comment]
Thanks for reporting this. I have run a couple of tests and I can see the issue in spark 2.2.1 for example
The issue appears to be fixed in spark 2.3.0 in my tests 
Example:
```
val stageMetrics = ch.cern.sparkmeasure.StageMetrics(spark)
stageMetrics.runAndMeasure(spark.sql("select id from range(10)").write.parquet("file:///tmp/deleteme2"))

...
sum(recordsWritten) => 10
sum(bytesWritten) => 3008 (2.0 KB)
...
```

**Comment by gilv** [comment]
@LucaCanali thanks for checking this. We did used Spark 2.2.1.. Good to know it's fixed in 2.3.0
